### PR TITLE
Fix: Extend the timeout of admin request

### PR DESCRIPTION
**Pull request by squarY**
Fix: Extend the timeout of admin request from 1min to 5min and add more logs for handling fetch failed request.

### CODE REVIEW

File: contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java [hunk 1]
```diff
@@ -120,9 +120,9 @@ public void run() throws Exception {
         }
 
         AdminClientConfig adminConfig = new AdminClientConfig().setMaxConnectionsPerNode(cluster.getNumberOfNodes())
-                                                               .setAdminConnectionTimeoutSec(15)
+                                                               .setAdminConnectionTimeoutSec(60)
                                                                .setMaxBackoffDelayMs(maxBackoffDelayMs)
-                                                               .setAdminSocketTimeoutSec(60);
+                                                               .setAdminSocketTimeoutSec(60 * 5); // Admin request timeout: 5min
```
Comment (arunthirupathi) [inline contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java]: Can you leave a comment saying that HdfsFailedFetchLock could take long time when the Name Node is in a different data center. So expanding it to 5 minutes.

File: src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java [hunk 2]
```diff
@@ -2033,9 +2033,11 @@ private Message handleFetchFailure(VAdminProto.HandleFetchFailureRequest handleF
         } else {
             FailedFetchLock distributedLock = null;
             try {
+                logger.info("Try to acqure a distributed lock.");
                 distributedLock = FailedFetchLock.getLock(voldemortConfig, new Props(extraInfoProperties));
 
                 distributedLock.acquireLock();
+                logger.info("Distributed lock acquired.");
```
Comment (arunthirupathi) [inline src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java]: I believe the lock acquired message belongs in the *Lock class, along with the time taken.

**Comment by arunthirupathi** [comment]
Left some minor comments, But feel free to merge this in first. Address the comments later.

File: contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java [hunk 3]
```diff
@@ -120,9 +120,9 @@ public void run() throws Exception {
         }
 
         AdminClientConfig adminConfig = new AdminClientConfig().setMaxConnectionsPerNode(cluster.getNumberOfNodes())
-                                                               .setAdminConnectionTimeoutSec(15)
+                                                               .setAdminConnectionTimeoutSec(60)
```
Comment (gaojieliu) [inline contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java]: The default connection timeout is 60s, we don't need to specify it if we want to use this value:
https://github.com/squarY/voldemort/blob/master/src/java/voldemort/client/protocol/admin/AdminClientConfig.java#L17

Comment (squarY) [inline contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java]: Will fix

File: src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java (hunk 2 above)
Comment (squarY) [inline src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java]: Will do

**Pull request merged**